pkgs/tools/misc/unicode/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, fetchurl, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  name = "unicode-${version}";
+  version = "2.5";
+
+  src = fetchFromGitHub {
+    owner = "garabik";
+    repo = "unicode";
+    rev = "v${version}";
+    sha256 = "0vg1zshlzgdva8gzw6fya28fc4jhypjkj743x3q0yabx6934k0g2";
+  };
+
+  ucdtxt = fetchurl {
+    url = http://www.unicode.org/Public/11.0.0/ucd/UnicodeData-11.0.0d1.txt;
```

Comment:
I think it is better not to depend on not-yet-released Unicode data. I guess this file will be renamed `UnicodeData.txt` in the future, and this URL will break.

Related: there was proposal at #29885 to move Unicode data to the separate package that other packages may depend on.

Reply:
You are right, I'll change that to the final 10.0.0 version.